src/physics/em/detail/BetheHeitler.hh

```diff
@@ -7,51 +7,84 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include "base/Assert.hh"
 #include "base/Macros.hh"
 #include "base/Types.hh"
+#include "random/RngEngine.hh"
+#include "physics/base/ModelInterface.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/PhysicsTrackView.hh"
 #include "physics/base/Types.hh"
+#include "base/StackAllocator.hh"
+#include "physics/material/MaterialTrackView.hh"
+#include "BetheHeitlerInteractor.hh"
 
 namespace celeritas
 {
-template<MemSpace M>
-struct ModelInteractRefs;
-
 namespace detail
 {
 //---------------------------------------------------------------------------//
 /*!
- * Device data for creating a BetheHeitlerInteractor.
+ * Model interactor kernel launcher
  */
-struct BetheHeitlerPointers
+template<MemSpace M>
+struct BetheHeitlerLauncher
```

Comment:
Since this file no longer contains a bunch of related "BetheHeitler" data, maybe we should rename it to `BetheHeitlerLauncher.hh` and update the generator script? We could have the generator script include both `BetheHeitler.hh` (for the definitions of `BetheHeitlerHostRef` and `BetheHeitlerDeviceRef`, which could live in `BetheHeitlerData` or whatever) and `BetheHeitlerLauncher.hh`/